docs/reference/manifest/supportssharedfolders.md

```diff
@@ -0,0 +1,21 @@
+# SupportsSharedFolders element
+
+Defines whether the Outlook add-in is available in delegate scenarios. The **SupportsSharedFolders** element is a child element of [DesktopFormFactor](desktopformfactor.md). It is set to *false* by default.
+
+> [!IMPORTANT]
+> This element is only available in the [Outlook add-ins Preview Requirement Set](../objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) against Exchange Online. Add-ins that use this element aren't allowed in the Office Store or Centralized Deployment.
```

Comment:
Same suggestion as made previously -- rephrase the final sentence of the note to:

Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment.